python/tank/bootstrap/cached_configuration.py

```diff
@@ -194,6 +194,18 @@ def update_configuration(self):
 
         # copy the configuration into place
         try:
+
+            # first do a quick validation check that the descriptor payload looks like a config
```

Comment:
a config without a core folder is not valid and this check helps falling back more gracefully and reporting a better error message back to the user.